tsdb/store.go

```diff
@@ -237,7 +237,11 @@ func (s *Store) loadShards() error {
 				}
 
 				shard := NewShard(s.databaseIndexes[db], path, s.EngineOptions)
-				shard.Open()
+				err = shard.Open()
+				if err != nil {
+					s.Logger.Printf("failed to open shard %d: %s", shardID, err)
```

Comment:
This feels like a real error here, I think we should actually return the error, not log it and continue.  if you can't load a shard something bad happened, and likely is going to have something else bad happen if we let it continue.  @pauldix thoughts?